### DISCUSSION TITLE
Welcome to Resum-Resume Discussions!

### DISCUSSION THREAD

**Discussion by GarimaSingh0109** [GarimaSingh0109/Resum-Resume#43] [Announcements]
Hey Guys I want you all to work on this website but follow a color pallet don't clutter things
The color pallet for this project will be -[https://coolors.co/palette/231942-5e548e-9f86c0-be95c4-e0b1cb](url)

**Comment by rishyym0927** [comment]
[https://coolors.co/palette/231942-5e548e-9f86c0-be95c4-e0b1cb](https://github.com/GarimaSingh0109/Resum-Resume/discussions/url) This link is not working

**Reply by GarimaSingh0109** [reply]
@rishyym0927 its working try copy pasting it

**Comment by soumyaranjan4446** [comment]
Hi Mam, I want to contribute to your project. Are there any issues labeled with Hacktoberfest and Gssoc exd  2024 that have no asignees. if yes then let me know . I can make changes in the font , layout and overall front end of the homepage.
